composer.json

```diff
@@ -135,6 +134,7 @@
         "fabpot/goutte": "^3.2",
         "friendsofphp/php-cs-fixer": "^2.16",
         "fzaninotto/faker": "^1.9",
+        "league/flysystem-memory": "^1.0",
```

Comment:
why did you moved this ine require-dev part ? It seems used in prod env too

Reply:
@kevin-verschaeve, it seems to be used only on test, no?
![image](https://user-images.githubusercontent.com/4582866/94280180-662cd400-ff4d-11ea-9503-5ca99b6cc7b3.png)

Reply:
my bad, misread the package name !